### ISSUE TITLE
[Update Request 2023]

### ISSUE THREAD

**Issue by joeyoropesa-dev** [Strencher/BdBrowser#37]
Can this plugin be updated? It's 2023 from the time of writing this message so I think that if we don't get any updates, this project is officially EOL.

Anyway without the update, this extension is useless because BetterDiscord is updated their API, manifest and plugins

**Comment by fowlis** [comment]
I don't know if its been officially stated anywhere, but I believe Strencher has no plans to continue this project.

**Comment by joeyoropesa-dev** [comment]
> I don't know if its been officially stated anywhere, but I believe Strencher has no plans to continue this project.

Yeah - I just found someone who created a fork that is in development so that this project is continue to be alive by one wonderful person who taked this project via fork feature on Discord and modded it to work in 2023.

**Comment by TERIHAX** [comment]
> > I don't know if its been officially stated anywhere, but I believe Strencher has no plans to continue this project.
> 
> Yeah - I just found someone who created a fork that is in development so that this project is continue to be alive by one wonderful person who taked this project via fork feature on Discord and modded it to work in 2023.

@admin-elvistrujanovic can you give me a link to that fork?

**Comment by joeyoropesa-dev** [comment]
> > > I don't know if its been officially stated anywhere, but I believe Strencher has no plans to continue this project.
> > 
> > Yeah - I just found someone who created a fork that is in development so that this project is continue to be alive by one wonderful person who taked this project via fork feature on Discord and modded it to work in 2023.
> 
> @admin-elvistrujanovic can you give me a link to that fork?

https://github.com/tsukasa/BdBrowser
